src/omni_epd/displays/inky_display.py

```diff
@@ -108,27 +119,34 @@ def __init__(self, deviceName, config):
         self.width = self._device.width
         self.height = self._device.height
 
+        # get colors from the inky lib (won't be used normally as inky does conversion)
+        self.palette_filter = deviceObj.DESATURATED_PALETTE
+
     @staticmethod
     def get_supported_devices():
         result = []
 
         # python libs for this might not be installed - that's ok, return nothing
         if(InkyImpressionDisplay.check_module_installed('inky')):
             # if passed return list of devices
-            result = [f"{InkyDisplay.pkg_name}.impression"]
+            result = [f"{InkyImpressionDisplay.pkg_name}.impression"]
 
         return result
 
     def _display(self, image):
-        self._device.set_image(image)
+
+        # no palette adjustments when color as the Inky lib does them from the image
+        if(self.mode == 'bw'):
+            image = self._filterImage(image)
+
+        self._device.set_image(image.convert("RGB"), saturation=self._getfloat_device_option('saturation', .5))  # .5 is default from Inky lib
         self._device.show()
 
     def clear(self):
-        from inky.inky_uc8159 import CLEAN
-
+        inkyDev = self.load_display_driver(self.pkg_name, 'inky_uc8159')
         for _ in range(2):
             for y in range(self.height - 1):
                 for x in range(self.width - 1):
-                    self._device.set_pixel(x, y, CLEAN)
+                    self._device.set_pixel(x, y, inkyDev.CLEAN)

```

Comment:
```suggestion
                    self._device.set_pixel(x, y, self._device.CLEAN)
```
This is defined in the Inky class as well, so it doesn't need to be loaded again.